service/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/mailgun/service/repo"
+	"net/http"
+	"time"
+)
+
+var (
+	port   string
+	dbConn *repo.DB
+)
+
+func main() {
+	parseArguements()
+
+	dbConn, dbError := repo.GetDB()
+	defer dbConn.Session.Close(context.Background())
+	if dbError != nil {
+		fmt.Println("msg", "Service failed to obtain database connection", "err", dbError)
+		panic("Service failed to obtain database connection")
```

Comment:
Is there a simpler way to exit the service?

Reply:
I think a better way would be to do multiple attempts to connect if it failed instead of quitting out. So maybe some kind of exponential backoff up to some many tries and then quit at that point. 
Extending this further you could just keep trying in a loop with that backoff timer and after so many failures send an alert out via slack/PD